src/main/java/org/accounting/system/repositories/modulators/AccessControlModulator.java

```diff
@@ -65,6 +65,20 @@ public List<E> getAllEntities() {
         return list("_id in ?1", List.of(entities));
     }
 
+    @Override
+    public List<ObjectId> getAllEntityIds() {
+
+        List<AccessControl> accessControlList = getAccessControlRepository().findAllByWhoAndCollection(getRequestInformation().getSubjectOfToken(), collection(), AccessControlPermission.READ);
+
+        List<ObjectId> entities = accessControlList
```

Comment:
I think, the assignment to variable entities is redundant.

Reply:
fixed